File: python/A02.py
# A02 Liner Search
# submission: https://atcoder.jp/contests/tessoku-book/submissions/38326123

n, x = map(int, input().split())
a = list(map(int, input().split()))

if any(aa == x for aa in a):
    print('Yes')
else:
    print('No')

File: rust/tessoku-book/src/bin/a02.rs
// A02 Linear Search
// submission: https://atcoder.jp/contests/tessoku-book/submissions/38759168

use proconio::input;

fn main() {
    input!{
        n: usize,
        x: usize,
        a:[usize; n],
    }
    let mut found: bool = false;
    for aa in &a {
        if *aa == x {
            found = true;
        }
    }
    if found {
        println!("Yes");
    } else {
        println!("No");
    }
}
